.circleci/config.yml

```diff
@@ -896,7 +896,18 @@ jobs:
           title: "browser"
           # Skip test_4gb_fail as it OOMs on the current bot
```

Comment:
Can add a similar comment here about these being smoke tests, leaving most for the 2G mode?